src/Common/src/Microsoft/Win32/SafeHandles/NtlmSecuritySafeHandles.cs

```diff
@@ -0,0 +1,271 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Security.Authentication;
+using System.Security.Authentication.ExtendedProtection;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace Microsoft.Win32.SafeHandles
+{
+    /// <summary>
+    /// Wrapper around a ntlm_buf*
+    /// </summary>
+    internal sealed class SafeNtlmBufferHandle : SafeHandle
+    {
+        public SafeNtlmBufferHandle()
+           : base(IntPtr.Zero, true)
+        {
+        }
+
+        public byte[] ToByteArray(int length, int offset)
```

Comment:
I find it weird when content methods are put on SafeHandle types.

If there's only one or two callers, I'd just inline this.  If there are more, I'd make it be a method in Interop.NetSecurity.

But that's just personal preference, I don't know that we have any strong guidance here.